### PR TITLE
Benchmark sdk suggestions

### PR DESCRIPTION
**Pull request by thiago-aixplain**

### CODE REVIEW

File: .pre-commit-config.yaml [hunk 1]
```diff
@@ -0,0 +1,18 @@
...
+        types: [python]
+        always_run: true
+  
+  - repo: https://github.com/psf/black
+    rev: 22.10.0
+    hooks:
+      - id: black
+        language_version: python3
+        args: # arguments to configure black
+          - --line-length=128
```
Comment (krishnadurai) [inline .pre-commit-config.yaml:18]: Thanks for setting this up

File: aixtend/factories/model_factory.py [hunk 2]
```diff
@@ -59,16 +57,22 @@ def create_model_from_id(cls, model_id: str) -> Model:
...
-        return model
-
+        try:
+            url = f"{cls.backend_url}/sdk/inventory/models/{model_id}"
+            headers = {"Authorization": f"Token {cls.api_key}", "Content-Type": "application/json"}
+            r = _request_with_retry("get", url, headers=headers)
+            resp = r.json()
+            model = cls._create_model_from_response(resp)
+            return model
+        except:
```
Comment (krishnadurai) [inline aixtend/factories/model_factory.py:67]: Please correct to `except Exception as e`

**Review by krishnadurai — COMMENTED** [review]
@thiago-aixplain just a minor comment.

**Review by shreyasXplain — APPROVED** [review]
Everything look okay. Will merge model tests to the new format

**Pull request merged**